news/2019-10-12-beatmap-pack-management-update-#01.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Beatmap Pack Management: Update #1"
+date: 2019-10-13 15:30:00 +0000
+---
+
+![](/wiki/shared/news/banners/beatmap-packs.jpg)
```

Comment:
above the header you can write a short paragraph that will be shown on the news listing and front page
![](https://up.clayton.cc/95a9f8.png)

strongly recommend doing that cuz every other news post in the last year or something has one

Reply:
Done.